# English (United States) translations for Synse Server.
# Copyright (C) 2019 Vapor IO
# This file is distributed under the same license as the Synse Server
# project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2019.
#
msgid ""
msgstr ""
"Project-Id-Version: Synse Server VERSION\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2019-11-15 14:09-0500\n"
"PO-Revision-Date: 2019-11-15 14:09-0500\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language: en_US\n"
"Language-Team: en_US <LL@li.org>\n"
"Plural-Forms: nplurals=2; plural=(n != 1)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 2.7.0\n"

#: synse_server/cache.py:84
msgid "caching transaction"
msgstr ""

#: synse_server/cache.py:107
msgid "adding alias to cache"
msgstr ""

#: synse_server/cache.py:142
msgid "updating the device cache"
msgstr ""

#: synse_server/cache.py:152
msgid "no plugins found when updating device cache"
msgstr ""

#: synse_server/cache.py:159
msgid "getting devices from plugin"
msgstr ""

#: synse_server/cache.py:177
msgid "failed to get device(s)"
msgstr ""

#: synse_server/cache.py:200
msgid "looking up device ID in cache"
msgstr ""

#: synse_server/cache.py:214
msgid "device ID not found in cache - checking for alias"
msgstr ""

#: synse_server/errors.py:52
msgid "an unexpected error occurred"
msgstr ""

#: synse_server/errors.py:76
msgid "invalid user input"
msgstr ""

#: synse_server/errors.py:88
msgid "resource not found"
msgstr ""

#: synse_server/errors.py:100
msgid "device action not supported"
msgstr ""

#: synse_server/errors.py:112
msgid "error processing the request"
msgstr ""

#: synse_server/plugin.py:76
msgid "registering new plugin"
msgstr ""

#: synse_server/plugin.py:80
msgid "application metrics enabled: registering gRPC interceptor"
msgstr ""

#: synse_server/plugin.py:106
msgid "plugin with id already registered - skipping"
msgstr ""

#: synse_server/plugin.py:109
msgid "successfully registered plugin"
msgstr ""

#: synse_server/plugin.py:124
msgid "loading plugins from configuration"
msgstr ""

#: synse_server/plugin.py:131 synse_server/plugin.py:137
msgid "loading plugin from config"
msgstr ""

#: synse_server/plugin.py:157
msgid "failed plugin discovery via Kubernetes"
msgstr ""

#: synse_server/plugin.py:174
msgid "refreshing plugin manager"
msgstr ""

#: synse_server/plugin.py:186
msgid "failed to register configured plugin - will attempt re-registering later"
msgstr ""

#: synse_server/plugin.py:200
msgid "failed to register discovered plugin - will attempt re-registering later"
msgstr ""

#: synse_server/plugin.py:206
msgid "plugin manager refresh complete"
msgstr ""

#: synse_server/plugin.py:283
msgid "marking plugin as active"
msgstr ""

#: synse_server/plugin.py:290
msgid "marking plugin as inactive"
msgstr ""

#: synse_server/server.py:85
msgid "initializing synse server"
msgstr ""

#: synse_server/server.py:90
msgid "loaded config"
msgstr ""

#: synse_server/server.py:97
msgid "configured logger"
msgstr ""

#: synse_server/server.py:104
msgid "created server directories on filesystem"
msgstr ""

#: synse_server/server.py:110
msgid "running synse server"
msgstr ""

#: synse_server/server.py:120
msgid "registering tasks with application"
msgstr ""

#: synse_server/server.py:125
msgid "application performance metrics enabled (/metrics)"
msgstr ""

#: synse_server/server.py:135
msgid "SSL configured for Synse Server"
msgstr ""

#: synse_server/server.py:137
msgid "running server without SSL (no key/cert configured)"
msgstr ""

#: synse_server/server.py:142
msgid "serving API endpoints"
msgstr ""

#: synse_server/tasks.py:18
msgid "adding task"
msgstr ""

#: synse_server/tasks.py:28
msgid "task: rebuilding device cache"
msgstr ""

#: synse_server/tasks.py:36
msgid "task: failed to rebuild device cache"
msgstr ""

#: synse_server/api/http.py:26
msgid "processing request"
msgstr ""

#: synse_server/api/websocket.py:23
msgid "new websocket connection"
msgstr ""

#: synse_server/api/websocket.py:40
msgid "failed to load payload"
msgstr ""

#: synse_server/api/websocket.py:121
msgid "running message handler for websocket"
msgstr ""

#: synse_server/api/websocket.py:127
msgid "error loading websocket message"
msgstr ""

#: synse_server/api/websocket.py:131
msgid "websocket handler: got message"
msgstr ""

#: synse_server/api/websocket.py:135
msgid "error generating websocket response"
msgstr ""

#: synse_server/api/websocket.py:199
msgid "processing websocket request"
msgstr ""

#: synse_server/api/websocket.py:204
msgid "websocket request cancelled"
msgstr ""

#: synse_server/api/websocket.py:207
msgid "error processing websocket request"
msgstr ""

#: synse_server/api/websocket.py:422
msgid "read stream stop request received - terminating stream tasks"
msgstr ""

#: synse_server/api/websocket.py:436
msgid "websocket raised ConnectionClosed - terminating read stream"
msgstr ""

#: synse_server/cmd/config.py:13 synse_server/cmd/info.py:18
#: synse_server/cmd/plugin.py:20 synse_server/cmd/plugin.py:60
#: synse_server/cmd/plugin.py:84 synse_server/cmd/read.py:62
#: synse_server/cmd/read.py:119 synse_server/cmd/read.py:157
#: synse_server/cmd/read.py:247 synse_server/cmd/scan.py:28
#: synse_server/cmd/tags.py:18 synse_server/cmd/test.py:13
#: synse_server/cmd/transaction.py:18 synse_server/cmd/transaction.py:65
#: synse_server/cmd/version.py:13 synse_server/cmd/write.py:21
#: synse_server/cmd/write.py:61
msgid "issuing command"
msgstr ""

#: synse_server/cmd/plugin.py:101
msgid "failed to get plugin health"
msgstr ""

#: synse_server/cmd/read.py:66
msgid "no tags specified, reading with no tag filter"
msgstr ""

#: synse_server/cmd/read.py:74 synse_server/cmd/read.py:98
msgid "error while issuing gRPC request: read"
msgstr ""

#: synse_server/cmd/read.py:78 synse_server/cmd/read.py:105
msgid "got readings"
msgstr ""

#: synse_server/cmd/read.py:85 synse_server/cmd/scan.py:55
msgid "parsing tag groups"
msgstr ""

#: synse_server/cmd/read.py:124 synse_server/cmd/write.py:28
#: synse_server/cmd/write.py:68
msgid "plugin not found for device {}"
msgstr ""

#: synse_server/cmd/read.py:133
msgid "error while issuing gRPC request: read device"
msgstr ""

#: synse_server/cmd/read.py:161
msgid "getting cached readings for plugin"
msgstr ""

#: synse_server/cmd/read.py:168
msgid "error while issuing gRPC request: read cache"
msgstr ""

#: synse_server/cmd/read.py:201
msgid "running Stream thread"
msgstr ""

#: synse_server/cmd/read.py:210
msgid "stream thread cancelled"
msgstr ""

#: synse_server/cmd/read.py:215
msgid "error while issuing gRPC request: read stream"
msgstr ""

#: synse_server/cmd/read.py:220
msgid "cancelling reading stream"
msgstr ""

#: synse_server/cmd/read.py:258
msgid "executing callback to cancel read stream threads"
msgstr ""

#: synse_server/cmd/read.py:270
msgid "collecting streamed readings..."
msgstr ""

#: synse_server/cmd/scan.py:36
msgid "forced scan: rebuilding device cache"
msgstr ""

#: synse_server/cmd/scan.py:40
msgid "failed to rebuild device cache"
msgstr ""

#: synse_server/cmd/scan.py:44
msgid "getting devices with no tag filter"
msgstr ""

#: synse_server/cmd/scan.py:49
msgid "failed to get all devices from cache"
msgstr ""

#: synse_server/cmd/scan.py:67
msgid "failed to get devices from cache"
msgstr ""

#: synse_server/cmd/scan.py:80
msgid "sorting devices"
msgstr ""

#: synse_server/cmd/scan.py:99
msgid "got devices"
msgstr ""

#: synse_server/cmd/tags.py:32
msgid "filtering ID tags"
msgstr ""

#: synse_server/cmd/tags.py:36
msgid "filtering tags by namespace"
msgstr ""

#: synse_server/cmd/tags.py:39
msgid "got tags"
msgstr ""

#: synse_server/cmd/write.py:43
msgid "error while issuing gRPC request: async write"
msgstr ""

#: synse_server/cmd/write.py:84
msgid "error while issuing gRPC request: sync write"
msgstr ""

#: synse_server/discovery/kubernetes.py:23
msgid "plugin discovery via Kubernetes is disabled"
msgstr ""

#: synse_server/discovery/kubernetes.py:35
msgid "plugin discovery via Kubernetes is enabled"
msgstr ""

#: synse_server/discovery/kubernetes.py:49
msgid "found no configured endpoints for plugin discovery via Kubernetes"
msgstr ""

#: synse_server/discovery/kubernetes.py:83
msgid "found no configured labels for plugin discovery via Kubernetes Endpoints"
msgstr ""

#: synse_server/discovery/kubernetes.py:100
msgid "listing Kubernetes endpoint"
msgstr ""

#: synse_server/discovery/kubernetes.py:108
msgid "discovered matching Endpoint"
msgstr ""

#: synse_server/discovery/kubernetes.py:111
msgid "parsing EndpointSubset"
msgstr ""

#: synse_server/discovery/kubernetes.py:117
msgid "no addresses for EndpointSubset - skipping"
msgstr ""

#: synse_server/discovery/kubernetes.py:123
msgid "collecting available addresses for EndpointSubset"
msgstr ""

#: synse_server/discovery/kubernetes.py:127
msgid "parsing EndpointAddress"
msgstr ""

#: synse_server/discovery/kubernetes.py:133
msgid "address has no target_ref - skipping"
msgstr ""

#: synse_server/discovery/kubernetes.py:138
msgid "address is not a Pod address - skipping"
msgstr ""

#: synse_server/discovery/kubernetes.py:147
msgid "no IPs found for EndpointSubset - skipping"
msgstr ""

#: synse_server/discovery/kubernetes.py:150
msgid "found IPs for EndpointSubset"
msgstr ""

#: synse_server/discovery/kubernetes.py:156
msgid "no ports for EndpointSubset - skipping"
msgstr ""

#: synse_server/discovery/kubernetes.py:162
msgid "found single port for EndpointSubset"
msgstr ""

#: synse_server/discovery/kubernetes.py:167
msgid "found multiple ports - search for port named \"http\""
msgstr ""

#: synse_server/discovery/kubernetes.py:169
msgid "found port"
msgstr ""

#: synse_server/discovery/kubernetes.py:171
msgid "skipping port - does not match"
msgstr ""

#: synse_server/discovery/kubernetes.py:175
msgid "found port name \"http\""
msgstr ""

#: synse_server/discovery/kubernetes.py:185
msgid "discovered plugin via Endpoint"
msgstr ""

#: synse_server/discovery/kubernetes.py:189
msgid "no plugins found via Kubernetes Endpoints"
msgstr ""

#: synse_server/discovery/kubernetes.py:191
msgid "found plugins via Kubernetes Endpoints"
msgstr ""

